qiskit_ibm_provider/utils/json_decoder.py

```diff
@@ -223,10 +226,16 @@ def target_from_server_data(
         converter = QobjToInstructionConverter(pulse_lib)
         for cmd in map(Command.from_dict, pulse_defaults["cmd_def"]):
             name = cmd.name
-            qubits = tuple(cmd.qubits)
-            if (name, qubits) in faulty_ops:
+            if name not in all_instructions or name not in prop_name_map:
+                logger.info(
+                    "Gate calibration for instruction %s is found "
+                    "in the PulseDefaults payload. However, this entry is not defined in "
+                    "the properties of Target. This calibration is ignored.",
```

Comment:
```suggestion
                    "Gate calibration for instruction %s is found "
                    "in the PulseDefaults payload. However, this entry is not defined in "
                    "the properties of Target. This calibration is ignored.",
```

Good catch. I don't think we need to give different error message, because this entry `cmd` is tied to some particular qubit, i.e. `Gate calibration for instruction %s on qubits %s` is actually found here and there is no calibration not aware of qubit. I think you just need to update the condition of the if-clause.

Reply:
Thanks! I united the messages.